CHANGELOG.md

```diff
@@ -75,6 +88,49 @@
   `BackgroundThreadsFactory()` member function, requiring anyone who derives
   from those classes to implement that function. This would typically only
   affect users who create mock clients for testing purposes.
+* fix(bigtable): multiple region tags on same line (#6051)
```

Comment:
These changes are not breaking, right? So we can split them with a `**OTHER CHANGES**:` header (or whatever we have used in the past)?

Also, #6051 was probably mislabeled, it is a cleanup and uninteresting to customers.